styleguide/template.md

```diff
@@ -285,6 +286,8 @@ Use three backticks (\`\`\`) + a language ID to apply language-specific color co
 |XAML|xaml|
 |XML|xml|
 
+The `csharp-interactive` name specifies the C# language, and the ability to run the samples from the browser. These snippets are compiled and executed in a Docker container, and the results of that program execution are displayed in the user's browser window.
```

Comment:
Should this paragraph be at the [section](https://github.com/dotnet/docs/blob/master/styleguide/template.md#code) above and mention `code-csharp-interactive`?

Reply:
I think it fits better here, after the table with all the language tags.